### DISCUSSION TITLE
Bundle analyzer not showing details but "webpack/loaders"

### DISCUSSION THREAD

**Discussion by joostmeijles** [vercel/next.js#23275] [Help]
I have several bundles that do not show internal details on whats inside. 
Instead the analyzer stops at "webpack/loaders".

See the example below:
![image](https://user-images.githubusercontent.com/4342282/112000165-025f1e80-8b1e-11eb-9575-bfe991306c45.png)

I am using:
- next: 10.0.9
- @next/bundle-analyzer: 10.0.9

Any clues?

**Comment by aphofstede** [comment]
@joostmeijles Did you ever solve this?

**Reply by kumarldh** [reply]
Not sure if it helps, you have to enable source map file generation when creating a build...

**Reply by aphofstede** [reply]
Thanks. Regrettably, adding `productionBrowserSourceMaps: true,` to the next config does not seem to make a difference
